dora/src/vm/namespaces.rs

```diff
@@ -69,6 +69,28 @@ impl NamespaceData {
         }
     }
 
+    pub fn init_modifiers(&mut self, sa: &SemAnalysis) {
+        #[allow(unused_variables)]
```

Comment:
Why?

Reply:
The AST hasn't `AnnotationUsages` yet.

Reply:
So AFAIU these init_modifiers aren't invoked atm, right? Also the code in there isn't really used atm. I don't think it makes sense to merge this PR as-is.

Why don't you first add a method `parse_annotation_usages` to the parser first that returns `AnnotationUsages`? AFAICS this method doesn't exist yet either. You could return both `Modifiers` and `AnnotationUsages` from that method to not break already existing code. Then switch element-by-element from `Modifiers` to `AnnotationUsages`.